contracts/badger-bridge/BadgerBridgeAdapter.sol

```diff
@@ -239,10 +270,34 @@ contract BadgerBridgeAdapter is OwnableUpgradeable, ReentrancyGuardUpgradeable {
             IBridgeVault(args._vault).token().safeApprove(args._vault, amount);
         }
 
-        IBridgeVault(args._vault).depositFor(args._user, amount);
+        if (args._mintIbbtc) {
+            //emit balanceCheckBefore(IERC20(args._vault).balanceOf(address(this)));
+            IBridgeVault(args._vault).depositFor(address(this), amount);
+            uint256 balance = IERC20(args._vault).balanceOf(address(this));
+            //emit balanceCheckAfter(balance);
+            
+            _mintIbbtc(args._vault, args._user, balance);
+        } else {    
+            IBridgeVault(args._vault).depositFor(args._user, amount);
+        }
+
         return true;
     }
 
+    function _mintIbbtc(address _vault, address _user, uint256 _amount) internal {
+        uint256 poolId = vaultToPoolId[_vault];
+        uint256 mintAmount = 0;
+        bytes32[] memory dummy;
```

Comment:
nit: rename to `unused`  or something to signal that it's an unused/deprecated param and add a comment for what it was for/why it's deprecated.